apps/3000-home/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "@module-federation/3000-home",
+  "version": "0.0.1",
+  "dependencies": {
+    "@nrwl/workspace": "14.7.3",
+    "fast-glob": "^3.2.12",
```

Comment:
`fast-glob` and `typescript` are redundant prod dependencies in demo apps.

Also I'm not sure that `@nrwl/workspace` should be placed in prod dependencies. It looks like it should be in dev dependencies

Reply:
@pavandv wrote:

`package.json` files are redundant for applications in `apps` folder. I'll be removing them in my next commit.

NX uses top-level package.json to manage all the dependencies needed for applications. Only libraries with NX Dev tools can get their own `package.json` file.

For this reason alone, I don't think I can resolve this comment you mentioned

> apps/3002-checkout should use old outdated `antd@4.20.0`
> It helps to check that module resolution with different `antd` versions works correctly

Reply:
In such a case, can we remove @nrwl/next from demo apps?
Testing different versions of the same package are more important than NX features.

Reply:
I have figured it a way to resolve the issue.. I'm working on it.

Reply:
Fixed 👍